lib/src/main/java/com/diffplug/spotless/npm/NpmResourceHelper.java

```diff
@@ -122,4 +123,20 @@
 			throw ThrowingEx.asRuntime(e);
 		}
 	}
+
+	static String md5(File file) {
+		return md5(readUtf8StringFromFile(file));
+	}
+
+	static String md5(String fileContent) {
+		MessageDigest md = ThrowingEx.get(() -> MessageDigest.getInstance("MD5"));
```

Comment:
@nedtwigg I'm pretty sure we can safely ignore this warning, as the hashing is merely used as a way to compact the package json content so it can be used for a folder name.
Alternatively I could switch to another way of compacting, e.g. [CRC-32](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/zip/CRC32.html)?

Reply:
I agree, MD5 is no problem for this usecase.